### ISSUE TITLE
Jaws reads out "values" of opening faucets

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/beers-law-lab#393]
**Device**
Samsung Laptop
**OS**
Win 11
**Browser**
Chrome + Jaws
**Problem Description**
For https://github.com/phetsims/qa/issues/1295.
When using Jaws to use the faucets the screen reader will read out values, even though this slider doesn't have specific values, just amounts of opening.
**Visuals**

https://github.com/user-attachments/assets/c8f8e3e0-9190-4e5d-b7a0-38ab2eed136b
